kDrive/Resources/en.lproj/Localizable.strings

```diff
@@ -2123,7 +2123,7 @@
 "trashActionRestoreFileOriginalPlace" = "Recover to the original location";
 
 /* loco:67a9ef7e628ce12f3209f422 */
-"trashAutoClearDescription" = "Trash is cleaned automatically every 30 days.";
+"trashAutoClearDescription" = "Trash is cleaned automatically every 30 days.\\n\nWould you like to increase this period?";
```

Comment:
Why is there a "\\n" in the string? Is it visible in app? Creating a new line in Loco is enough